apps/frontend/src/lib/imports/update.ts

```diff
@@ -0,0 +1,48 @@
+import { ImportDataType } from "@types";
+import { UserData } from "@/lib/storage/types";
+
+// Any way to generalize?
+export const updateUserDataFromImportData = async (
+  userData: UserData,
+  type: ImportDataType,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  data: any
+): Promise<UserData> => {
+  switch (type) {
+    case ImportDataType.STRAVA_PREVIOUS_MONTH_RUN_DISTANCE:
+      const value1: number = data;
+      return {
+        ...userData,
+        strava: {
+          previousMonthRunDistance: {
+            value: value1,
+            lastUpdated: new Date(),
+          },
+        },
+      };
+    case ImportDataType.GITHUB_LANNA_CONTRIBUTIONS:
+      const value2: number = data;
+      return {
+        ...userData,
+        github: {
+          lannaCommits: {
+            value: value2,
+            lastUpdated: new Date()
+          },
+        },
+      };
+    case ImportDataType.GITHUB_CONTRIBUTIONS_LAST_YEAR:
+      const value3: number = data;
+      return {
+        ...userData,
+        github: {
+          annualCommits: {
+            value: value3,
+            lastUpdated: new Date()
```

Comment:
This should have a corresponding backup, will add in the next PR. This import type is not currently fetched (it was not added to the import options global var), will double check.